src/operator/src/insert.rs

```diff
@@ -238,8 +202,34 @@ impl Inserter {
             ..Default::default()
         });
 
-        let tasks = self
-            .group_requests_by_peer(requests.clone())
+        // Mirror requests for source table to flownode
+        match self.mirror_flow_node_requests(&requests).await {
+            Ok(flow_requests) => {
+                let node_manager = self.node_manager.clone();
+                let flow_tasks = flow_requests.into_iter().map(|(peer, inserts)| {
+                    let node_manager = node_manager.clone();
+                    common_runtime::spawn_write(async move {
```

Comment:
how about `spawn_bg` to avoid impact front-ground write

Reply:
Looks good.